### PR TITLE
Fix StyleCI configuration

### PR DESCRIPTION
**Pull request by manuerumx**
For https://github.com/Medology/stdcheck.com/issues/9429

Fix StyleCI configuration error:

```
Config Issue
The provided fixer 'simplified_null_return' cannot be disabled unless it was already enabled by your preset.
```

### CODE REVIEW

File: .idea/php.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
   </component>
   <component name="PhpUnit">
     <phpunit_settings>
-      <PhpUnitSettings load_method="CUSTOM_LOADER" custom_loader_path="$PROJECT_DIR$/vendor/autoload.php" />
+      <PhpUnitSettings custom_loader_path="$PROJECT_DIR$/vendor/autoload.php" />
```
Comment (Chekote) [inline .idea/php.xml]: Why was this changed? It seems unrelated to StyleCI.

Comment (manuerumx) [reply .idea/php.xml]: Removed.
